Userland/Libraries/LibELF/DynamicObject.h

```diff
@@ -65,19 +65,11 @@ class DynamicObject : public RefCounted<DynamicObject> {
         FlatPtr value() const { return m_sym.st_value; }
         size_t size() const { return m_sym.st_size; }
         unsigned index() const { return m_index; }
-#if ARCH(I386)
```

Comment:
Is it sensible to mark those places with `#if ARCH(X86_64) || ARCH(AARCH64)` and error with unsupported architecture in the other case?